dll/win32/comdlg32/lang/cdlg_Pt.rc

```diff
@@ -246,7 +246,7 @@ FONT 8, "MS Shell Dlg"
     LTEXT           "&Tamanho:", stc3, 179, 3, 32, 9
     COMBOBOX        cmb3, 179, 13, 32, 76, CBS_OWNERDRAWFIXED | CBS_HASSTRINGS | CBS_DISABLENOSCROLL |
                                            WS_VSCROLL | WS_TABSTOP | CBS_SIMPLE | CBS_SORT
-    DEFPUSHBUTTON   "&OK", IDOK, 218, 6, 50, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
+    DEFPUSHBUTTON   "OK", IDOK, 218, 6, 50, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
```

Comment:
correct for the coose-font-dlg, it doesn't have an accelerator here on XPSP3:
![image](https://github.com/user-attachments/assets/910d684e-b904-44cd-9116-bd72b0a84d79)